asdf_astropy/resources/schemas/wcs/slicedwcs-1.1.0.yaml

```
@@ -0,0 +1,44 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://astropy.org/schemas/astropy/wcs/slicedwcs-1.1.0"
+
+title: Represents the SlicedLowLevelWCS object
+
+description: >-
+  The SlicedLowLevelWCS class is a wrapper class for WCS that applies slices
+  to the WCS, allowing certain pixel and world dimensions to be retained or
+  dropped.
+  It manages the slicing and coordinate transformations while preserving
+  the underlying WCS object.
+
+allOf:
+  - type: object
+    properties:
+      wcs:
+        anyOf:
+          - tag: "tag:astropy.org:astropy/wcs/wcs-1*"
+          - tag: "tag:stsci.edu:gwcs/wcs-*"
```

This also needs to support all the ndcube wrappers as well:


```suggestion
          - tag: "tag:stsci.edu:gwcs/wcs-*"
          - tag: "tag:sunpy.org:ndcube/resampledwcs-*"
          - tag: "tag:sunpy.org:ndcube/ndcubesequence-*"
          - tag: "tag:sunpy.org:ndcube/reorderedwcs-*"
          - tag: "tag:sunpy.org:ndcube/compoundwcs-*"
```

Although as I said on #296 (https://github.com/astropy/asdf-astropy/pull/296#discussion_r2336218911) maybe we should just be allowing any object here?

---

Would you help me understand how is ndcubesequence a wcs? I'm not familiar with the class.

---

Whoops, copy paste error.

---

I'm leaning towards adding each of these (with 1.x wildcards: `wcs-1.*`). There's probably a somewhat academic discussion to be had about leaking in-memory API structure into the serialization format since as you mentioned here the goal is to allow anything that is API compliant (which doesn't jive with a constraint on data structure).
I'm also ok with allowing any object here since as noted the intended constraint is on API and not on structure.
I'll leave this one up to you. I'm ok with either the suggestion you added or allowing any object.